js/Items.js

```diff
@@ -0,0 +1,99 @@
+        {id: 51, name: 'Paragon of Calamity', value: -50, section: 'taken', table: 'increased', elements: [1,2,3]},
+        {id: 52, name: 'Rite of Ruin', value: -6, section: 'taken', table: 'increased', elements: [0,1,2,3,4]},
+        {id: 53, name: 'Aspect of Carnage', value: 10, section: 'taken', table: 'increased', elements: [0,1,2,3,4]},
+        {id: 54, name: 'Unrelenting', value: -8, section: 'taken', table: 'increased', elements: [1,2,3]},
+        {id: 55, name: 'Outmatch and Outlast', value: -10, section: 'taken', table: 'increased', elements: [0]},
+        {id: 56, name: 'Blood in the Eyes', value: -6, section: 'taken', table: 'increased', elements: [0,1,2,3,4]},
+        {id: 57, name: 'Conqueror', value: -6, section: 'taken', table: 'increased', elements: [0,1,2,3,4]},
+        {id: 58, name: 'Headsman', value: -50, section: 'taken', table: 'increased', elements: [0]},
+        {id: 59, name: 'Nature\'s Boon', value: -8, section: 'taken', table: 'increased', elements: [1,2,3]},
+        {id: 60, name: 'Born in the Shadows', value: -6, section: 'taken', table: 'increased', elements: [0,1,2,3,4]},
+        {id: 61, name: 'Weave the Arcane', value: -8, section: 'taken', table: 'increased', elements: [0,1,2,3,4]},
+        {id: 62, name: 'Elementalist (Ascendant)', value: -50, section: 'taken', table: 'increased', elements: [1,2,3]},
+        {id: 63, name: 'Inquisitor (Ascendant)', value: -8, section: 'taken', table: 'increased', elements: [0,1,2,3,4]},
+        {id: 64, name: 'Berserker (Ascendant)', value: 5, section: 'taken', table: 'increased', elements: [0,1,2,3,4]},
+        {id: 65, name: 'Slayer (Ascendant)', value: -50, section: 'taken', table: 'increased', elements: [0]},
+        {id: 66, name: 'Champion (Ascendant)', value: -5, section: 'taken', table: 'increased', elements: [0,1,2,3,4]},
+
+        // More Damage Taken
+        {id: 67, name: 'Kintsugi', value: -20, section: 'taken', table: 'more', elements: [0,1,2,3,4]},
+        {id: 68, name: 'Arctic Armour (Phys)', value: -13, section: 'taken', table: 'more', elements: [0]},
+        {id: 69, name: 'Arctic Armour (Fire)', value: -12, section: 'taken', table: 'more', elements: [1]},
+
+        // Damage Taken From Mana Before Life
+        {id: 70, name: 'Mind Over Matter', value: 30, section: 'shift', table: 'mana', elements: [0,1,2,3,4]},
+        {id: 71, name: 'Divine Guidance', value: 10, section: 'shift', table: 'mana', elements: [0,1,2,3,4]},
+        {id: 72, name: 'Cloak of Defiance', value: 20, section: 'shift', table: 'mana', elements: [0,1,2,3,4]},
+        {id: 73, name: 'The Aylardex', value: 6, section: 'shift', table: 'mana', elements: [0,1,2,3,4]},
+        {id: 74, name: 'Mind of the Council', value: 30, section: 'shift', table: 'mana', elements: [3]},
+        {id: 75, name: 'Damage taken from Mana Corruption', value: 6, section: 'shift', table: 'mana', elements: [0,1,2,3,4]}
+      ].sort(function (a, b) {
```

Comment:
This should probably be a map eventually (faster lookups when I implement encoding modifiers as IDs), and we then return map.values.sort or something.